### PR TITLE
use ubuntu 16.04 instead of 14.04

### PR DESCRIPTION
**Pull request by misiek-sf**
change machine executor to one that is recommended by circle ci.

`circleci/classic:201808-01` image is using 5 years old release which got into EOL at 30th of April

but the main reason for that is to bump docker version to `18.09.3` which is bundled in `ubuntu-1604:201903-01`. that version of docker contains (most notable) build kit with the possibility to use ssh agent forwarding [release notes](https://docs.docker.com/develop/develop-images/build_enhancements/)

### CODE REVIEW

**Comment by afaq-sf** [comment]
@wprater @codin-mgm Do we have any other teams at MGM using the VPN orb that would need to be aware of this change?

File: vpn.yml [hunk 1]
```diff
@@ -4,9 +4,7 @@ executors:
   aws:
     machine:
       enabled: true
-      # The `classic:201808-01` image has a later version of the aws cli
-      # which is important for ECS deployments
-      image: circleci/classic:201808-01
+      image: ubuntu-1604:201903-01
```
Comment (afaq-sf) [inline vpn.yml:7]: @misiek-sf  Can you please add a brief explanation in the PR description of why this change is necessary for unblocking CCI?

**Review by afaq-sf — CHANGES_REQUESTED** [review]

File: vpn.yml (hunk 1 above)
Comment (misiek-sf) [reply vpn.yml:7]: done

**Review by afaq-sf — APPROVED** [review]
